inference-engine/src/preprocessing/arm_neon/ie_preprocess_gapi_kernels_neon.cpp

```diff
@@ -708,6 +624,28 @@ void calcRowLinear_8UC1(uint8_t* dst[],
     }
 }
 }  // namespace neon
+
+template void chanToPlaneRowImpl(neon_tag, const uint8_t* in, int chan, int chs, uint8_t* out, const int length);
+template void chanToPlaneRowImpl(neon_tag, const float*   in, int chan, int chs, float  * out, const int length);
+
+template void nv12ToRgbRowImpl(neon_tag, const uint8_t** y_rows, const uint8_t* uv_row, uint8_t** out_rows, const int buf_width);
+
+template void i420ToRgbRowImpl(neon_tag, const uint8_t** y_rows, const uint8_t* u_row,
+                               const uint8_t* v_row, uint8_t** out_rows, const int buf_width);
+
+template void splitRowImpl(neon_tag, const uint8_t* in, std::array<uint8_t*, 2>& outs, const int length);
+template void splitRowImpl(neon_tag, const float* in, std::array<float*, 2>& outs, const int length);
+template void splitRowImpl(neon_tag, const uint8_t* in, std::array<uint8_t*, 3>& outs, const int length);
```

Comment:
OPTIONAL (sorry that i did not suggest in prev PR):
 what do you think about MACRO ?
```
#define FORCE_INST_(tag)   \
template void splitRowImpl(tag,... ); \
template void splitRowImpl(tag,... ); \
...
```

then 
```
//in `neon.cpp`
FORCE_INST(neon_tag);

//in `inavx.cpp`
FORCE_INST(avx);
```